lib/api/unittest/CDataFrameTrainBoostedTreeClassifierRunnerTest.cc

```diff
@@ -115,4 +123,23 @@ BOOST_AUTO_TEST_CASE(testWriteOneRow) {
     }
 }
 
+BOOST_AUTO_TEST_CASE(testWriteOneRow_DependentVariableIsInt) {
```

Comment:
Coding standards (and others below)
```suggestion
BOOST_AUTO_TEST_CASE(testWriteOneRowDependentVariableIsInt) {
```